fxa/core.py

```diff
@@ -278,12 +285,7 @@ def get_email_status(self):
         return resp
 
     def verify_email_code(self, code):
-        body = {
-            "uid": self.uid,
-            "code": code,
-        }
-        url = "/recovery_email/verify_code"
-        self.apiclient.post(url, body)  # note: not authenticated
+        return self.client.verify_email_code(self.uid, code) # note: not authenticated
```

Comment:
Of course! Just a question.

Sorry, but I have to ask a general GitHub PR-workflow question: Can I actually push force to my branch and change the PR or should I create a new one, once I have added this space and a test for the method?

(I guess it would be best to change the commit and not add a commit that just adds a space, right?)